modules/ROOT/pages/exportword.adoc

```diff
@@ -36,15 +36,45 @@ tinymce.init({
   selector: 'textarea',
   plugins: 'exportword',
   toolbar: 'exportword',
+// Below option is only required when using the cloud-based Export to Word plugin from Tiny.Cloud.
+// Avoid setting it up during the trial period.
+  exportword_token_provider: () => {
+    return fetch('http://localhost:3000/jwt', { // specify your token endpoint
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+    }).then(response => response.json());
```

Comment:
What is the expected JSON structure? Is it supposed to be encoded? FYI https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-token-structure

Reply:
@tiny-ben-tran you mean encrypted?